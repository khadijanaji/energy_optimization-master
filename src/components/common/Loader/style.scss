.center-block {
  display: flex;
  justify-content: center;
  align-items: center;
  margin: auto;
  padding-top: 50%;
}