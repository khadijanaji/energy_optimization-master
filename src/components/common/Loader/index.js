// @flow

import CircularProgress from "@material-ui/core/CircularProgress";
import React from "react";

import "./style.scss";

const Loader = () => (
  <div className="center-block">
    <CircularProgress />
  </div>
);

export default Loader;
