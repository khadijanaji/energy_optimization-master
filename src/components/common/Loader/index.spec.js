/* globals describe it expect */

import {shallow} from "enzyme";
import React from "react";

import Loader from "./";

describe("Component", () => {
  describe("Loader Render", () => {
    it("Should match snapshot", () => {
      const wrapper = shallow(<Loader />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
