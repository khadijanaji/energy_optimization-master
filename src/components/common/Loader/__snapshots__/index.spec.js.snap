// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Loader Render Should match snapshot 1`] = `
<div
  className="center-block"
>
  <WithStyles(CircularProgress) />
</div>
`;
